modules/ecs_service_fargate_elb/variables.tf

```diff
@@ -86,3 +88,32 @@ variable "target_group_arn" {
   description = "Target of an Amazon ALB (Load Balancer), as given by a load balancer."
 }
 
+variable "datadog_api_key" {
```

Comment:
@rafops forgive my Nulogy Deployer noobishness, but is there a reason that this variable also appears in `module_groups/ecs_fargate_with_elb/variables.tf`?

The situation is the same for `datadog_agent_version`.

Reply:
Hey @sskirby, that's an authentic question. What is happening here is a chain of modules passing down these variables: `app_worker -> module_groups/ecs_fargate_with_elb -> modules/ecs_service_fargate_elb/variables.tf`